extend.php

```diff
@@ -39,7 +39,13 @@
         ->serializeToForum('categories.parentRemoveDescription', 'askvortsov-categories.parent-remove-description', 'boolval')
         ->serializeToForum('categories.parentRemoveStats', 'askvortsov-categories.parent-remove-stats', 'boolval')
         ->serializeToForum('categories.parentRemoveLastDiscussion', 'askvortsov-categories.parent-remove-last-discussion', 'boolval')
-        ->serializeToForum('categories.childBareIcon', 'askvortsov-categories.child-bare-icon', 'boolval', true),
+        ->serializeToForum('categories.childBareIcon', 'askvortsov-categories.child-bare-icon', 'boolval', true)
+        ->serializeToForum('categories.widgetHeader', 'askvortsov-categories.widget-header', 'boolval', false)
+        ->serializeToForum('categories.widgetRight', 'askvortsov-categories.widget-right', 'boolval', false)
```

Comment:
Why not default these to true? If someone has installed a widget extension, presumably they want widgets?

Reply:
Not entirely, I though it was better to turn them off by default with the original categories page. No one seems to have complained about not having widgets on the page. The original didn't have header, footer or right side widget. Only there was a bug for allowing the widget to show on the left sidebar, which I accidently fixed. This PR was to make it almost like before, but with options